files/base.rb

```diff
@@ -101,7 +105,7 @@ def full_description
 Occurrences:  #{@event['occurrences']}
 
 Team: #{team_name}
-Host: #{@event['client']['name']}
+Host: #{client_display_name}
```

Comment:
At eat24, the client name is the instance ID, so it has value independent of the display name and the host's address.  For us and others, it may still be valuable to include the client name in the ticket.  Something like:

```Client Name: #{@event['client']['name']```